server/src/main/java/org/opensearch/script/JodaCompatibleZonedDateTime.java

```diff
@@ -427,120 +401,7 @@ public ZonedDateTime withZoneSameInstant(ZoneId zone) {
         return dt.withZoneSameInstant(zone);
     }
 
-    @Deprecated
```

Comment:
Are you also looking at backporting this to 2.0?
If not, can we start curating a breaking changes meta issue for 3.0?  
This is a breaking change and there could be plugins in the wild using these methods.

Reply:
Yes, sorry, forgot to add the labels.